extensions/wikia/StaffLog/StaffLog.events.php

```diff
@@ -64,8 +64,8 @@ static public function eventlogWFPublicStatusChange( $cityStatus, $cityId, $reas
 		global $wgUser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app